aiida/backends/djsite/db/subtests/migrations.py

```diff
@@ -59,19 +58,16 @@ def setUp(self):
         except Exception:
             # Bring back the DB to the correct state if this setup part fails
             self._revert_database_schema()
-            raise
```

Comment:
I think we should keep this `raise` (the same is also done in SQLA)

Reply:
I will add it in PR #2402